packages/phoenix-evals/src/phoenix/evals/utils.py

```diff
@@ -174,3 +175,22 @@ def _default_openai_function(
 def printif(condition: bool, *args: Any, **kwargs: Any) -> None:
     if condition:
         tqdm.write(*args, **kwargs)
+
+
+def get_audio_format_from_base64(enc_str: str) -> Literal["mp3", "wav"]:
+    audio_bytes = base64.b64decode(enc_str)
+
+    if len(audio_bytes) >= 3:
+        # Check for MP3 (ID3 tag or frame sync bytes)
+        if audio_bytes[0:3] == b"ID3":
+            return "mp3"
+        elif audio_bytes[0] == 0xFF and (audio_bytes[1] & 0xE0 == 0xE0):  # MPEG sync
+            return "mp3"
+
+    if len(audio_bytes) >= 12:
+        # Check for WAV (RIFF header with WAVE format)
+        if audio_bytes[0:4] == b"RIFF" and audio_bytes[8:12] == b"WAVE":
+            return "wav"
+
+    # If no match, raise an error
+    raise ValueError("Unsupported audio format. Only wav and mp3 are supported.")
```

Comment:
can you keep all of the formats in this detector? handle the unsupported formats directly in the `OpenAIModel` class